server/repository/src/db_diesel/store_row.rs

```diff
@@ -11,9 +12,19 @@ table! {
         code -> Text,
         site_id -> Integer,
         logo -> Nullable<Text>,
+        store_mode -> crate::db_diesel::store_row::StoreModeMapping,
     }
 }
 
+#[derive(DbEnum, Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(test, derive(strum::EnumIter))]
+#[DbValueStyle = "SCREAMING_SNAKE_CASE"]
+pub enum StoreMode {
+    Store,
+    Dispensary,
+    DrugRegistration,
```

Comment:
I only did it because it was in the test... I talked to Andrei about this on Friday, and he said only to add Store and Dispensary... what do you think? (Can remove the test for it as well)

Reply:
No fine with me! yes maybe remove the test then if it doesn't test anything else (or change the mode in the test)